test/e2e_node/security_context_test.go

```diff
@@ -42,158 +39,6 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		podClient = f.PodClient()
 	})
 
-	ginkgo.Context("when pod PID namespace is configurable [Feature:ShareProcessNamespace][NodeAlphaFeature:ShareProcessNamespace]", func() {
```

Comment:
since this is still beta, is there any need to actually move this? we can't promote it to conformance

Reply:
ack. I'll restrict this PR to just hostPID